spark-cassandra-connector-embedded/src/main/scala/com/datastax/spark/connector/embedded/SparkTemplate.scala

```diff
@@ -27,6 +27,7 @@ object SparkTemplate {
   val defaultConf = new SparkConf(true)
     .set("spark.cassandra.connection.host", getHost(0).getHostAddress)
     .set("spark.cassandra.connection.port", getPort(0).toString)
+    .set("spark.ui.enabled", "false")
```

Comment:
actually - either SPARK_LOCAL_IP set to 127.0.0.1 or disable UI - otherwise UI will try to use 0.0.0.0 which in general is wrong in case of Jenkins workers and there are plenty of errors.